### DISCUSSION TITLE
How to save the image locally?

### DISCUSSION THREAD

**Discussion by kevin4dhd** [NarHakobyan/awesome-nest-boilerplate#294] [Q&A] [answered]
I don't want to use AWS to save the images, I want to save it on the same server where the project is hosted, is there an example or section where I should change? I really need it I would be very grateful if you help me

**Comment by iojanis** [comment]
This is essential for me as well... I'm going to have a look at it. But maybe there's someone who could give some advice?

**Reply by alaamh** [reply]
did you get any luck with this?

**Comment by NarHakobyan** [comment, marked as answer]
You can use [multer implementation for Nestjs](https://docs.nestjs.com/techniques/file-upload#default-options), with specifying `dest` field.
